data/tlg0018/tlg001/tlg0018.tlg001.opp-grc1.xml

```diff
@@ -103,966 +104,963 @@
-πάντ’ οἰκονομεῖσθαι καὶ πρυτανεύεσθαι θέμις.</p></div>
-
-<div type="textpart" subtype="chapter" n="12">
-<p>ἀλλ’ ὅ γε μέγας Μωυσῆς  ἀλλοτριώτατον τοῦ  <milestone unit="altpage2" n="p. 3M."/> ὁρατοῦ νομίσας εἶναι τὸ ἀγένητον — πᾶν γὰρ τὸ
-<lb n="15"/> αἰσθητὸν ἐν γενέσει καὶ μεταβολαῖς οὐδέποτε κατὰ ταὐτὰ ὄν — τῷ μὲν
-ἀοράτῳ καὶ νοητῷ προσένειμεν ὡς ἀδελφὸν καὶ συγγενὲς ἀιδιότητα, τῷ
-δ’ αἰσθητῷ γένεσιν οἰκεῖον ὄνομα ἐπεφήμισεν. ἐπεὶ οὖν ὁρατός τε καὶ
-αἰσθητὸς ὅδε ὁ κόσμος, ἀναγκαίως ἂν εἴη καὶ γενητός· ὅθεν οὐκ ἀπὸ
-σκοποῦ καὶ τὴν γένεσιν ἀνέγραψεν αὐτοῦ μάλα σεμνῶς θεολογήσας.</p>
-<note type="footnote">1 ἡ ἐπιστήμη VM καὶ κρεῖττον M, καὶ κρεῖττον ΑΒ αὑτὸ τὸ καλὸν καὶ αὐτὸ τὸ
-ἀγαθόν Β τἀγαθὸν Α 2 παθητικὸν V Eus κεινηθὲν V 3. 4 τόνδε τὸν
-κόσμον] τὸν δὲ κόσμον FGHR 4 ὃν οἱ φάσκοντες V Eus: ὃν om. ceteri ἀγέννητον
-(om. ὡς ἔστιν) R Eus 5 τῶν εἰς εὐσέβειαν ἡκόντων FGHR ἀποτεμνόμενοι R
+<div type="textpart" subtype="section" n="11">
+<p><lb n="10"/> πρὸς δὲ τὸ μὴ γεγονὸς οἰκείωσις οὐδεμία τῷ μὴ πεποιηκότι. ἀπεριμάχητον δὲ δόγμα καὶ ἀνωφελὲς ἀναρχίαν ὡς ἐν πόλει κατασκευάζον τῷδε
+τῷ κόσμῳ τὸν ἔφορον ἢ βραβευτὴν ἢ δικαστὴν οὐκ ἔχοντι, ὑφ’ οὗ
+πάντ’ οἰκονομεῖσθαι καὶ πρυτανεύεσθαι θέμις.</p></div>
+
+<div type="textpart" subtype="section" n="12">
+<p>ἀλλ’ ὅ γε μέγας Μωυσῆς  ἀλλοτριώτατον τοῦ  <milestone unit="altpage2" n="p. 3M."/> ὁρατοῦ νομίσας εἶναι τὸ ἀγένητον — πᾶν γὰρ τὸ
+<lb n="15"/> αἰσθητὸν ἐν γενέσει καὶ μεταβολαῖς οὐδέποτε κατὰ ταὐτὰ ὄν — τῷ μὲν
+ἀοράτῳ καὶ νοητῷ προσένειμεν ὡς ἀδελφὸν καὶ συγγενὲς ἀιδιότητα, τῷ
+δ’ αἰσθητῷ γένεσιν οἰκεῖον ὄνομα ἐπεφήμισεν. ἐπεὶ οὖν ὁρατός τε καὶ
+αἰσθητὸς ὅδε ὁ κόσμος, ἀναγκαίως ἂν εἴη καὶ γενητός· ὅθεν οὐκ ἀπὸ
+σκοποῦ καὶ τὴν γένεσιν ἀνέγραψεν αὐτοῦ μάλα σεμνῶς θεολογήσας.</p>
+<note type="footnote">1 ἡ ἐπιστήμη VM καὶ κρεῖττον M, καὶ κρεῖττον ΑΒ αὑτὸ τὸ καλὸν καὶ αὐτὸ τὸ
+ἀγαθόν Β τἀγαθὸν Α 2 παθητικὸν V Eus κεινηθὲν V 3. 4 τόνδε τὸν
+κόσμον] τὸν δὲ κόσμον FGHR 4 ὃν οἱ φάσκοντες V Eus: ὃν om. ceteri ἀγέννητον
+(om. ὡς ἔστιν) R Eus 5 τῶν εἰς εὐσέβειαν ἡκόντων FGHR ἀποτεμνόμενοι R
 ἑ
-6 ποιητὴν καὶ πατέρα R αἱρεῖ (sic) V, αἱρεῖ M, ἐρεῖ G, ἱστορεῖ Η, οἶδεν R
+6 ποιητὴν καὶ πατέρα R αἱρεῖ (sic) V, αἱρεῖ M, ἐρεῖ G, ἱστορεῖ Η, οἶδεν R
 γ
-ὁ λόγος R 7 ὁ γὰρ R ἐκγόνων M καὶ ὅ R 8 καὶ βλαβερὰ om.
+ὁ λόγος R 7 ὁ γὰρ R ἐκγόνων M καὶ ὅ R 8 καὶ βλαβερὰ om.
 Eus 9 τὰ δ’ ὅσα MFGHR, τὰ δ’ ὅσ’ ABP, ὅσα δὲ Eus κατὰ π. τ. V:
-κατὰ om. ceteri 10 γεγονὼς F οἰκίωσις V, ὁμοίωσις M 10. 11 ἀπεριμάχητον
-FGH: περιμάχητον ceteri 11 κατασκευάζον (ον ex ων corr.) V:
-ἐστὶ MABP, κατασκευάζειν FGH Eus τῶνδε V, τῶ δὲ M 12 ὡς
-τὸν ἔφορον V, sed ὡς rursus deletum ἔφορον] φόβον Eus 13 πάντ’ V Eus:
-τὰ πάντ’ M, τὰ πάντα ceteri γε om. Eus Μωυσῆς VF: Μώσης ceteri
-15 τὰ αὐτὰ codd. ὃν] ὄντα M 15. 16 κατὰ ταῦτα ὄντων μὲν ἀοράτων καὶ
-νοητῶν V 16 συγγενῆ Eus 17 γένεσιν om. Eus καὶ οἰκεῖον Η, τὸ
-οἰκεῖον V εἴτ’ ἐφήμισεν V ἀόρατός V 18 ἀναγκαῖος VR
+κατὰ om. ceteri 10 γεγονὼς F οἰκίωσις V, ὁμοίωσις M 10. 11 ἀπεριμάχητον
```

Comment:
There seems to be Latin interspersed here.

Reply:
that's the app crit, which has Latin interspersed.